brain/task_executor.py

```
@@ -496,6 +746,149 @@ async def _execute_mcp(
+            logger.info(f"[TaskExecutor] Using explicit plugin_entry_id for trigger: {plugin_entry_id}")
+        # send trigger
+        logger.info(f"[TaskExecutor] POST to plugin trigger {trigger_endpoint} with body: {trigger_body}")
+        try:
+            import httpx
+            async with httpx.AsyncClient(timeout=5.0) as client:
+                r = await client.post(trigger_endpoint, json=trigger_body)
+                # Treat 2xx as accepted. plugin_server may synchronously execute and return executed_entry
+                if 200 <= r.status_code < 300:
+                    try:
+                        data = r.json()
+                    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)